app/views/layouts/application.html.haml

```diff
@@ -15,6 +15,8 @@
     %header#header
       %h1 Stranger Teams
       %ul#nav
+
+        %li= link_to "Home", root_path
```

Comment:
This is fine for now, but let's remember to make the entire header a link back home.